src/main/java/com/datadoghq/sketch/ddsketch/DDSketch.java

```diff
@@ -388,6 +385,20 @@ public static DDSketch fast(double relativeAccuracy) {
         );
     }
 
+    /**
+     * Constructs a fast instance of {@code DDSketch}, with optimized ingestion speed, at the cost of higher memory
+     * usage, but avoids allocating large ranges of zeros.
+     *
+     * @param relativeAccuracy the relative accuracy guaranteed by the sketch
+     * @return a fast instance of {@code DDSketch}
+     */
+    public static DDSketch fastPaginated(double relativeAccuracy) {
```

Comment:
I merged #28, which moves preset sketches to `DDSketches` and tries to use more explicit method names. Should we go for `bitwiseLinearlyInterpolatedPaginated` maybe?

Reply:
Sounds good. I just rebased, perhaps we can leave it as is because the constructors allow composition for now, and, as discussed, this store doesn't come without costs for very small sketches.